components/scream/src/physics/rrtmgp/CMakeLists.txt

```diff
@@ -76,8 +76,8 @@ target_include_directories(scream_rrtmgp SYSTEM PUBLIC
 
 # Ensure RRTMGP lookup tables are present in the data dir
 set (RRTMGP_TABLES
-  init/rrtmgp-data-sw-g224-2018-12-04.nc
-  init/rrtmgp-data-lw-g256-2018-12-04.nc
+  init/rrtmgp-data-sw-g112-210809.nc #rrtmgp-data-sw-g224-2018-12-04.nc
```

Comment:
any reason to keep the commented out part?

Reply:
Nope, good catch will fix.

Reply:
Fixed.